project_closing/models/project.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+from odoo import models, api
+
+
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    @api.multi
+    def toggle_active(self):
+        for record in self:
+            record.analytic_account_id.active = not record.active
```

Comment:
It's better to invoke toggle_active() method on record.analytic_account_id in order to keep consistency with the already existing logic (and eventually with other modules editing that functionality).